src/index.html

```diff
@@ -6,7 +6,41 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="reset">
+    <main class="container">
+      <div class="calendar calendar--start-day_tue calendar--m-leng_28">
```

Comment:
```suggestion
      <div 
        class="calendar calendar--start-day_tue calendar--month-length_28"
      >
```
Let's make it clear

Reply:
I got it. Thank you! My point was to shorten modifier name a little, but for next projects will keep in mind that better to work on readability than on shortness